pkg/core/client/client.go

```diff
@@ -1,15 +1,58 @@
 package client
 
 import (
+	"context"
+	"io"
+
 	rawclient "github.com/nspcc-dev/neofs-api-go/v2/rpc/client"
 	"github.com/nspcc-dev/neofs-node/pkg/network"
 	"github.com/nspcc-dev/neofs-sdk-go/client"
+	"github.com/nspcc-dev/neofs-sdk-go/container"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	"github.com/nspcc-dev/neofs-sdk-go/eacl"
+	"github.com/nspcc-dev/neofs-sdk-go/owner"
 )
 
 // Client is an interface of NeoFS storage
 // node's client.
 type Client interface {
-	client.Client
+	GetBalance(context.Context, *owner.ID, ...client.CallOption) (*client.BalanceOfRes, error)
+
+	PutContainer(context.Context, *container.Container, ...client.CallOption) (*client.ContainerPutRes, error)
+	GetContainer(context.Context, *cid.ID, ...client.CallOption) (*client.ContainerGetRes, error)
+	ListContainers(context.Context, *owner.ID, ...client.CallOption) (*client.ContainerListRes, error)
+	DeleteContainer(context.Context, *cid.ID, ...client.CallOption) (*client.ContainerDeleteRes, error)
+
+	EACL(context.Context, *cid.ID, ...client.CallOption) (*client.EACLRes, error)
+	SetEACL(context.Context, *eacl.Table, ...client.CallOption) (*client.SetEACLRes, error)
+
+	AnnounceContainerUsedSpace(context.Context, []container.UsedSpaceAnnouncement, ...client.CallOption) (*client.AnnounceSpaceRes, error)
+
+	EndpointInfo(context.Context, ...client.CallOption) (*client.EndpointInfoRes, error)
+	NetworkInfo(context.Context, ...client.CallOption) (*client.NetworkInfoRes, error)
+
+	PutObject(context.Context, *client.PutObjectParams, ...client.CallOption) (*client.ObjectPutRes, error)
+	DeleteObject(context.Context, *client.DeleteObjectParams, ...client.CallOption) (*client.ObjectDeleteRes, error)
+	GetObject(context.Context, *client.GetObjectParams, ...client.CallOption) (*client.ObjectGetRes, error)
+	HeadObject(context.Context, *client.ObjectHeaderParams, ...client.CallOption) (*client.ObjectHeadRes, error)
+	SearchObjects(context.Context, *client.SearchObjectParams, ...client.CallOption) (*client.ObjectSearchRes, error)
+	ObjectPayloadRangeData(context.Context, *client.RangeDataParams, ...client.CallOption) (*client.ObjectRangeRes, error)
+	HashObjectPayloadRanges(context.Context, *client.RangeChecksumParams, ...client.CallOption) (*client.ObjectRangeHashRes, error)
+
+	AnnounceLocalTrust(context.Context, client.AnnounceLocalTrustPrm, ...client.CallOption) (*client.AnnounceLocalTrustRes, error)
+	AnnounceIntermediateTrust(context.Context, client.AnnounceIntermediateTrustPrm, ...client.CallOption) (*client.AnnounceIntermediateTrustRes, error)
+
+	CreateSession(context.Context, uint64, ...client.CallOption) (*client.CreateSessionRes, error)
+
+	Raw() *rawclient.Client
+
+	Conn() io.Closer
+}
+
+// MultiAddressClient is an interface of the
+// Client that supports multihost work.
+type MultiAddressClient interface {
```

Comment:
Lets try to avoid this type. If some package needs such an interface, let it define it itself.

Reply:
so you mean to reorganize interfaces? should it be done in that PR?

i ask because there are no new interfaces: this PR's `Client` is ex `Client` from SDK and this PR's `MultiAddressClient` is ex `Client` from the same package of the node repo, so just renaming/moving, no new interfaces in fact

Reply:
I got it, then do not rename and abstract from the fact that the client is multiaddress.

Reply:
We've decided with @carpawell to reconsider client interfaces later.